CITATION.cff

```diff
@@ -1,6 +1,6 @@
 title: STRchive
-version: 2.5.1
-date-released: "2025-07-08"
+version: 2.5.2
```

Comment:
Since there's a change to the HTT min pathogenic range, I think this should be a major version change. So 2.6.0.